File: Modul-2/Soal-4/PRAK204-2410817210008-AhmadUlyani.c
#include <stdio.h>

int main (){
    float jarijari, tinggi, volume, luas, keliling;
    scanf("%f %f", &jarijari, &tinggi);
    volume = 22.0/7.0 * jarijari * jarijari * tinggi;
    luas = 2 * 22.0/7.0 * jarijari * (jarijari + tinggi);
    keliling = 2 * 22.0/7.0 * jarijari;

    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    printf("\n");
    scanf("%f %f", &jarijari, &tinggi);
    volume = 22.0/7.0 * jarijari * jarijari * tinggi;
    luas = 2 * 22.0/7.0 * jarijari * (jarijari + tinggi);
    keliling = 2 * 22.0/7.0 * jarijari;
    printf("Volume = %.2f\n", volume);
    printf("Luas = %.2f\n", luas);
    printf("Keliling = %.2f\n", keliling);

    return 0;
}

File: Modul-2/Soal-4/PRAK204-2410817210008-AhmadUlyani.py
jarijari = float(input(""))
tinggi = float(input(""))
volume = 22/7 * jarijari * jarijari * tinggi
luas = 2 * 22/7 * jarijari * (jarijari + tinggi)
keliling = 2 * 22/7 * jarijari 

print(f'Volume = {volume:.2f}')
print(f'Luas = {luas:.2f}')
print(f'Keliling = {keliling:.2f}')

print("")

jarijari, tinggi = map(float, input("").split())
volume = 22/7 * jarijari * jarijari * tinggi
luas = 2 * 22/7 * jarijari * (jarijari + tinggi)
keliling = 2 * 22/7 * jarijari 

print(f'Volume = {volume:.2f}')
print(f'Luas = {luas:.2f}')
print(f'Keliling = {keliling:.2f}')